packages/react/src/CTAForm/CTAForm.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react'
+import clsx from 'clsx'
+import {FormControl, Button, ButtonProps, TextInput, TextInputProps, Checkbox} from '../'
+
+import styles from './CTAForm.module.css'
+
+type CTAFormProps = {
+  inputLabel: TextInputProps['leadingText']
+  inputType?: TextInputProps['type']
+  placeholder?: TextInputProps['placeholder']
+  buttonLabel: ButtonProps<'button'>['children']
+  checkboxLabel: React.ReactNode[] | React.ReactNode | string
+} & React.HTMLAttributes<HTMLFormElement>
```

Comment:
This is quite a closed API. Was there a reason for not going with a composable API here? I can't find the API exploration to see the wider conversation.